src/test/java/com/thealgorithms/sorts/BitonicSortTest.java

```diff
@@ -0,0 +1,10 @@
+package com.thealgorithms.sorts;
+
+public class BitonicSortTest extends SortingAlgorithmTest {
+    private final BitonicSort bitonicSort = new BitonicSort();
+
+    @Override
+    SortAlgorithm getSortAlgorithm() {
+        return bitonicSort;
```

Comment:
Why not simply? It should be rather cheap to create an instance of `BitonicSort`.
```suggestion
    @Override
    SortAlgorithm getSortAlgorithm() {
        return new BitonicSort();
```